scripts/all-tests.sh

```diff
@@ -13,7 +13,7 @@ set -e
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=4175
 export JSHINT_THRESHOLD=7550
-export SAFELINT_THRESHOLD=2700
+source scripts/safelint_thresholds.sh
```

Comment:
a better way might be to store the string in a .json file and read it from the file directly into the envar.